swift/ql/src/queries/Security/CWE-089/SqlInjection.qhelp

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+
+<p>
+If a database query (such as an SQL query) is built from user-provided data without sufficient sanitization, a user may be able to run malicious database queries.
+</p>
+
+</overview>
+<recommendation>
+
+<p>
+Most database connector libraries offer a way of safely embedding untrusted data into a query by means of query parameters or prepared statements. Use these features rather than building queries by string concatenation.
+</p>
+
+</recommendation>
+<example>
+
+<p>In the following example, an SQL query is prepared using string interpolation to directly include a user-controlled value <code>userControlledString</code> in the query. An attacker could craft the part they control to change the overall meaning of the SQL query.
```

Comment:
I hope this isn't adding too much overhead but could you kindly explain what's meant by "the part they control" (the attacker)? The reason I ask is because this paragraph feels like the central part of the whole piece and some feedback I have might or might not be relevant.

Reply:
It means the part of the concatenated string that comes from the user.  I agree it may be worth moving this bit to sooner in the document.

Reply:
I've added something very similar to this point to the opening paragraph of the doc.  There's a bit of repetition but that may not be a bad thing.  I've also rephrased "the part they control" to be a bit more explicit.